mixer/pkg/config/adapterInfoRegistry.go

```diff
@@ -33,32 +32,32 @@ type handlerBuilderValidator func(hndlrBuilder adapter.HandlerBuilder, t string)
 func newRegistry2(infos []adapter.InfoFn, hndlrBldrValidator handlerBuilderValidator) *adapterInfoRegistry {
 	r := &adapterInfoRegistry{make(map[string]*adapter.Info)}
 	for idx, info := range infos {
-		glog.V(6).Infof("registering [%d] %#v", idx, info)
+		log.Debugf("registering [%d] %#v", idx, info)
 		adptInfo := info()
 		if a, ok := r.adapterInfosByName[adptInfo.Name]; ok {
 			// panic only if 2 different adapter.Info objects are trying to identify by the
 			// same Name.
-			msg := fmt.Errorf("duplicate registration for '%s' : old = %v new = %v", a.Name, adptInfo, a)
-			glog.Error(msg)
+			msg := fmt.Sprintf("Duplicate registration for '%s' : old = %v new = %v", a.Name, adptInfo, a)
```

Comment:
Just a minor FYI: This was probably this way because of the go linter requirement for error messages to start with small caps.